index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BOSE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="shortcut icon" href="favicon.png" type="image/x-icon">
+  </head>
+  <body class="page">
+    <header class="header" id="home">
+      <nav class="navigation header__navigation">
+        <div class="logo navigation__logo">
+          <a href="/" class="logo__link navigation__item">
+          <img class="logo__image" src="./src/images/logo-bose.svg" alt="">
+          </a>
+        </div>
+        <div class="icons navigation__item navigation__icons">
+          <a href="tel:+1 234 555-55-55" class="phone">
+          <span class="phone__label">+1 234 555-55-55</span> 
+          <img class="phone__image" src="./src/images/icon-phone.svg" alt="">
+          </a>
+          
+          <a href="#menu" class="burger navigation__item icons__burger"><span class="burger__label">menu</span></a>
+        </div>
+      </nav>
+      <h1 class="title header__title title--header">
+        The world shades.<br>
+        Your music shines.
+      </h1>
+    </header>
+
+    <div class="menu" id="menu">
+      <nav class="navigation menu__navigation">
+        <a href="#" class="logo navigation__item"><img class="logo__image" src="./src/images/logo-bose.svg" alt=""></a>
+        <a href="#" class="navigation__item--hidden"><img class="phone__image" src="./src/images/icon-phone.svg"
```

Comment:
Please fix formatting code:

```suggestion
        <a href="#" class="navigation__item--hidden">
          <img class="phone__image" src="./src/images/icon-phone.svg" alt="">
        </a>
        <a href="#" class="close navigation__item">
          <img class="close__image" src="./src/images/icon-menu-close.svg" alt="">
        </a>
      </nav>
```